db/migrations/versions/62956d81eb2f_ingredients_db_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Ingredients db migration
+
+Revision ID: 62956d81eb2f
+Revises: 
+Create Date: 2023-11-09 19:13:03.944592
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '62956d81eb2f'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
If your alembic configuration is correct here you will see create table statement